src/main/scala/com/ironcorelabs/DeviceContext.scala

```diff
@@ -33,6 +33,15 @@ case class DeviceContext(
 }
 
 object DeviceContext {
+  def apply(context: jsdk.DeviceContext): DeviceContext =
```

Comment:
we should document this apply

Reply:
Sorry, I must have crossed my eyes here. I thought this was a DeviceContext apply that took the params to construct the case class.